tests/utils/fakes/fake_dataset_collection.py

```diff
@@ -17,8 +17,8 @@ def files(self) -> FileCollection:
 
 class FakeDatasetCollection(DatasetCollection):
 
-    def __init__(self, project_id, session):
-        DatasetCollection.__init__(self, project_id=project_id, session=session)
+    def __init__(self, project_id, session, team_id):
+        DatasetCollection.__init__(self, team_id = team_id, project_id=project_id, session=session)
```

Comment:
Hmmmmmmmm...not sure why this was never moved over to `super`. Mind making that change while you're here?

Reply:
Can do!